.github/workflows/karate.yml

```diff
@@ -0,0 +1,114 @@
+name: Integration Tests
+
+on:
+  schedule:
+    # Run the ci every day at 19:00
+    - cron:  '00 19 * * *'
+    
+jobs:
+  be2-scala:
+    name: Be2-Scala
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          distribution: "zulu"
+          java-version: "11"
+
+      - name: Setup repo
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0 # Shallow clones should be disabled for a better relevancy of Sonar analysis
```

Comment:
we're not running Sonar, though, right? The comment might be a bit outdated 😁

Reply:
This might be an artifact of a copy-paste 😅